src/checksums.ts

```diff
@@ -5,17 +5,15 @@ const httpc = new httpm.HttpClient('gradle/wrapper-validation-action')
 export async function fetchValidChecksums(
   allowSnapshots: boolean
 ): Promise<string[]> {
-  const all: object[] = await httpGetJsonArray(
-    'https://services.gradle.org/versions/all'
-  )
```

Comment:
Why remove the type?

Reply:
Because we can 😉 The other two assignments in this function also do not list the types. It seems more consistent to either list all or no types, and as having them does not really increase readability IMO, so I decided to remove them.